src/flow.h

```diff
@@ -76,6 +77,8 @@ typedef struct AppLayerParserState_ AppLayerParserState;
 #define FLOW_TOCLIENT_DROP_LOGGED         0x00004000
 /** alproto detect done.  Right now we need it only for udp */
 #define FLOW_ALPROTO_DETECT_DONE          0x00008000
+/** time machine is enabled on this flow */
+#define FLOW_TIMEMACHINE_ENABLED          0x00010000
```

Comment:
I ultimately decided on FLOW_CONTAINS_ALERTS as I didn't see a good way to indicate whether a flow actually contained an alert or not since the majority of alerts are tagged to the final packet that actually caused the alert to fire.  Once that packet is processed, there's no going back.  FLOW_CONTAINS_ALERTS is a bit more generic but exactly the condition that I'm looking for.  Thoughts?